docs/adr/stac-generation-ingestion-user-stories.md

```diff
@@ -0,0 +1,193 @@
+# MAAP STAC Metadata Generation and Ingestion User Stories
+
+This document captures the user stories for all use cases of the MAAP STAC metadata generation and ingestion pipeline (public and user-generated STACs).
+
+**Goal:** enumerate all common use-cases so that we can confidently design modular infrastructure that can serve all use-cases.
+
+## Use Case 1: DPS Job Outputs Cataloging in User-STAC
+
+### User Story 1A: Single DPS Job Output Cataloging
+
+**As a** researcher running a data processing job,  
+**I want to** have the outputs of my DPS job automatically cataloged in the user-STAC,  
+**So that** I can easily discover, access, visualize, and share my processed data products.
+
+**Acceptance Criteria:**
+
+- I can specify a collection ID (new or existing) for my job outputs
+- If I don't specify a collection, items are added to my default username-based collection
+- I control which outputs get indexed by generating a simple JSON with asset key/href pairs (e.g., `{"data": "output.tif", "log": "log.txt"}`)
```

Comment:
Why is it not just all files in the `output` directory? One good reason is that that this method would allow users to control the asset name for easier use later.

Reply:
Exactly. I think we need them to declare the asset keys so we don't do something like set the asset keys to file basenames and wind up with unique asset keys for each item.